data/sql/updates/pending_db_world/rev_1605908814314990400.sql

```diff
@@ -0,0 +1,2469 @@
+(11244, 'frFR', 'Ce que les vrykuls ont fait à nos éclaireurs est d’une sauvagerie épouvantable. C’est une tactique de terreur, pour saper notre détermination… pour que nous nous enfuyions par crainte de perdre la vie.$b$bÇa ne marchera pas.$b$bNous allons montrer à ces racailles ce que cela signifie de faire la guerre contre l’Alliance !', 18019),
+(11245, 'frFR', 'Excellent pointage, messire ! Il faudra que je me souvienne d’offrir une tournée aux canonniers du donjon.', 18019),
+(11246, 'frFR', 'Et bien, messire, le moins qu’on puisse dire, c’est que vous vous êtes chargé de ce boulot avec zèle.$b$b<Le sergent vous observe prudemment.>$b$bDe toute façon, si ça ne vous dérange pas, je vais récupérer ma machette. Elle a l’air d’être émoussée, et ça serait dommage que vous dérapiez et que vous me coupiez moi aussi en morceaux, hein.', 18019),
+(11247, 'frFR', 'Très bien, messire. Je n’y serais pas arrivé sans votre aide.$b$bJe suis sûr que le capitaine sera heureux d’entendre mon rapport une fois que nous en aurons terminé ici.', 18019),
+(11248, 'frFR', 'C’est un plaisir de faire votre connaissance, messire. Je suis impatient de travailler avec vous.$b$bÀ ce propos, nous avons beaucoup à faire.', 18019),
+(11249, 'frFR', 'Oui, mon fils, cela a l’air bien sinistre, et vous avez eu bien raison d’intervenir.$b$bMerci beaucoup de m’avoir apporté ce parchemin. J’ai hâte de l’étudier pour comprendre ce qu’il implique. Si je devais hasarder une hypothèse, je dirais que ces vrykuls ont trouvé un moyen de transcender la mort ; enfin, pour ceux qui en sont dignes.$b$bEn vainquant le thane des Mörghiver, vous avez prouvé qu’il n’en était pas digne. Si vous ne l’aviez pas tué, la Lumière seule sait ce qu’il serait devenu, et quelles choses terribles il aurait lâché sur nous.', 18019),
+(11250, 'frFR', 'That''s unbelievably good news! I knew that you could pull it off, &lt;name&gt;!$B$BAt least someone around here can produce results!$B$BWell, this calls for a reward. Go ahead, take your pick; the treasury is open to you.', 18019),
+(11251, 'frFR', 'Bon, z''êtes peut-être en retard, mais ça fait du bien de voir un visage ami.$b$bBon, maintenant, arrêtez de rester planté là, à bâiller aux corneilles… Y a du travail à faire !', 18019),
+(11252, 'frFR', 'Outstanding!$B$BThis could be the turn of luck we''ve all been hoping for.$B$BWell done, &lt;name&gt;!', 18019),
+(11255, 'frFR', 'Les soldats que vous avez sauvés avaient des informations intéressantes. Ces monstres s’appellent des « vrykuls ». La tribu qui s’est installée au village de Crâne-du-ver est dirigée par un mystique nommé Yanis, qui travaille pour un autre vrykul, un certain Ingvar le Pilleur.', 18019),
+(11260, 'frFR', 'Oui, c’est sinistre, et vous avez certainement eu raison.$b$bMerci de m’avoir remis ce parchemin, je vais l’étudier. Si je devais hasarder une hypothèse, je dirais que ces vrykuls ont un moyen de transcender la mort – ceux d’entre eux qui en sont jugés dignes, en tout
```

Comment:
```suggestion
INSERT INTO `quest_request_items_locale` (`ID`, `locale`, `CompletionText`, `VerifiedBuild`) VALUES 
(9483, 'frFR', 'Ce n\'est pas parce que je suis en pèlerinage que je dois abandonner tous les plaisirs de la vie. Cela fait des siècles que je n\'ai pas degusté un bon vin.$b$bJ\'en arriverais presque à tuer pour une bouteille d\'esprit-de-vin cénarien !$b$bJe sais, je sais, ce n\'est pas du tout la mode de boire quelque chose produit par les elfes de la nuit.', 18019);
```

Reply:
Not sure if this is correct as you suggested, better keep the original like before